lib/routes/util.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+exports.shouldSkip = function (requestOrReply) {
+    if (
+        !(
+            requestOrReply.context.config &&
```

Comment:
Questa può essere fatta senza if, è un po ridondante l'`if`

Reply:
Credo lo abbia fatto per aumentarne la leggibilità

Reply:
> This should be done without the `if`. It is redundant, WDYT?

What you had in mind?

Reply:
```js
export.shouldSkip = (requestOrReply) => !(
            requestOrReply.context.config &&
            requestOrReply.context.config.metrics &&
            requestOrReply.context.config.metrics.routeId
            );
```

Reply:
Ok 😉